frontend/src/app/modules/availability/new-slot/questions/questions.component.sass

```
@@ -17,15 +18,34 @@ $question-input-width: 337px
     display: flex
     flex-direction: column
 
+.flex-row
+    display: flex
+    flex-direction: row
+    justify-content: space-between
```

Use `.flex` class to create flex container with `flex-direction: row`. `flex-direction: row` is the default value for flex.

`justify-content: space-between` goes beyond the scope of responsibility of `.flex-row` class. Move it into separate class and add it to flex containers, which need to be space-between

```suggestion
.sb
    justify-content: space-between
```